src/Collection/Collection.router.ts

```diff
@@ -221,7 +221,7 @@ export class CollectionRouter extends Router {
     req: AuthRequest
   ): Promise<PaginatedResponse<FullCollection> | FullCollection[]> => {
     const { page, limit } = getPaginationParams(req)
-    const { assignee, status, sort, q, is_published } = req.query
+    const { assignee, status, sort, q, isPublished } = req.query
```

Comment:
Modify the param is_published to isPublished because the front end sends it in CamelCase format

Reply:
I believe the front is sending `is_published` as snake case in one of the flows. (Check the `toRemoteCollectionQueryParameters` function)

Reply:
You're right! let me revert these changes